src/plugin/noob_plugin.js

```diff
@@ -30,6 +30,12 @@ class NoobPluginVirtual extends EventEmitter {
     this.id = opts.id // not used but required for compatability with five
                       // bells connector.
     this.auth = opts.auth
+    this.prefix = opts.auth.prefix
```

Comment:
The noob should get its address/prefix from the nerd, that's what the getPrefix method does right now